check50.py

```diff
@@ -32,8 +33,8 @@ def main():
     parser.add_argument("files", nargs="*")
     parser.add_argument("-d", "--debug", action="store_true")
     parser.add_argument("-l", "--local", action="store_true")
-    parser.add_argument("-f", "--force", action="store_true")
     parser.add_argument("--log", action="store_true")
+    parser.add_argument("--no-autoupdate", action="store_true")
```

Comment:
Hm, am thinking `--update` is more natural, since it's more conventional to have to tell a program to update something, rather than it being assumed. And aliasing on IDE to `check50 --update` and on Offline to `check50 --local` feels like a nice parallel?